### DISCUSSION TITLE
Add `{% csrf %}` shorthand

### DISCUSSION THREAD

**Discussion by notramo** [martenframework/marten#194] [Ideas]
Since the CSRF token is mostly used in `<form>` elements in a hidden input, it would be useful to add a `{% csrf %}` template tag, which renders a hidden input:
```html
<input type=hidden name=csrftoken value="#{csrf_token}">
```

**Comment by ellmetha** [comment]
Just a friendly reminder that this isn't a bug. I'd appreciate it if we could use GitHub discussions to brainstorm ideas or propose new features instead.

I think adding a more explicit `csrf_input` template tag is something we could do indeed. This should be an easy addition to the framework. 👍
